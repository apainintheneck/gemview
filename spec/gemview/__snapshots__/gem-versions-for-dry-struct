[
  {
    "number": "1.6.0",
    "downloads_count": 7496385,
    "created_at": "2022-11-04 17:52:19 UTC"
  },
  {
    "number": "1.5.2",
    "downloads_count": 4062223,
    "created_at": "2022-10-19 08:43:17 UTC"
  },
  {
    "number": "1.5.1",
    "downloads_count": 33012,
    "created_at": "2022-10-17 14:11:44 UTC"
  },
  {
    "number": "1.5.0",
    "downloads_count": 3388,
    "created_at": "2022-10-15 05:40:04 UTC"
  },
  {
    "number": "1.4.0",
    "downloads_count": 8190875,
    "created_at": "2021-01-21 18:59:57 UTC"
  },
  {
    "number": "1.3.0",
    "downloads_count": 4171242,
    "created_at": "2020-02-10 14:58:39 UTC"
  },
  {
    "number": "1.2.0",
    "downloads_count": 1317595,
    "created_at": "2019-12-20 08:50:48 UTC"
  },
  {
    "number": "1.1.1",
    "downloads_count": 2274245,
    "created_at": "2019-10-13 11:14:16 UTC"
  },
  {
    "number": "1.1.0",
    "downloads_count": 33203,
    "created_at": "2019-10-07 09:21:30 UTC"
  },
  {
    "number": "1.0.0",
    "downloads_count": 1300165,
    "created_at": "2019-04-23 15:43:45 UTC"
  },
  {
    "number": "0.7.0",
    "downloads_count": 669886,
    "created_at": "2019-03-22 13:54:13 UTC"
  },
  {
    "number": "0.6.0",
    "downloads_count": 2239203,
    "created_at": "2018-10-24 08:37:28 UTC"
  },
  {
    "number": "0.5.1",
    "downloads_count": 615227,
    "created_at": "2018-08-11 11:47:24 UTC"
  },
  {
    "number": "0.5.0",
    "downloads_count": 320650,
    "created_at": "2018-05-03 14:48:15 UTC"
  },
  {
    "number": "0.4.0",
    "downloads_count": 1070735,
    "created_at": "2017-11-04 17:30:47 UTC"
  },
  {
    "number": "0.3.1",
    "downloads_count": 924916,
    "created_at": "2017-06-30 10:03:35 UTC"
  },
  {
    "number": "0.3.0",
    "downloads_count": 95748,
    "created_at": "2017-05-05 09:00:44 UTC"
  },
  {
    "number": "0.2.1",
    "downloads_count": 222226,
    "created_at": "2017-02-27 22:02:43 UTC"
  },
  {
    "number": "0.2.0",
    "downloads_count": 2953,
    "created_at": "2017-02-26 16:15:21 UTC"
  },
  {
    "number": "0.1.1",
    "downloads_count": 881809,
    "created_at": "2016-11-13 12:31:47 UTC"
  },
  {
    "number": "0.1.0",
    "downloads_count": 25330,
    "created_at": "2016-09-22 00:11:24 UTC"
  },
  {
    "number": "0.0.1",
    "downloads_count": 12314,
    "created_at": "2016-07-17 03:26:30 UTC"
  }
]