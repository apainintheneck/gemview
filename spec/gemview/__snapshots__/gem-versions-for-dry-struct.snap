---
- !ruby/object:Gemview::Gem::Version
  version: 1.6.0
  downloads: 7496385
  release_date: 2022-11-04
  ruby_version: ">= 2.7.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.5.2
  downloads: 4062223
  release_date: 2022-10-19
  ruby_version: ">= 2.7.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.5.1
  downloads: 33012
  release_date: 2022-10-17
  ruby_version: ">= 2.7.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.5.0
  downloads: 3388
  release_date: 2022-10-15
  ruby_version: ">= 2.7.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.4.0
  downloads: 8190875
  release_date: 2021-01-21
  ruby_version: ">= 2.5.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.3.0
  downloads: 4171242
  release_date: 2020-02-10
  ruby_version: ">= 2.4.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.2.0
  downloads: 1317595
  release_date: 2019-12-20
  ruby_version: ">= 2.4.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.1.1
  downloads: 2274245
  release_date: 2019-10-13
  ruby_version: ">= 2.4.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.1.0
  downloads: 33203
  release_date: 2019-10-07
  ruby_version: ">= 2.4.0"
- !ruby/object:Gemview::Gem::Version
  version: 1.0.0
  downloads: 1300165
  release_date: 2019-04-23
  ruby_version: ">= 2.4.0"
- !ruby/object:Gemview::Gem::Version
  version: 0.7.0
  downloads: 669886
  release_date: 2019-03-22
  ruby_version: ">= 2.3.0"
- !ruby/object:Gemview::Gem::Version
  version: 0.6.0
  downloads: 2239203
  release_date: 2018-10-24
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.5.1
  downloads: 615227
  release_date: 2018-08-11
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.5.0
  downloads: 320650
  release_date: 2018-05-03
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.4.0
  downloads: 1070735
  release_date: 2017-11-04
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.3.1
  downloads: 924916
  release_date: 2017-06-30
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.3.0
  downloads: 95748
  release_date: 2017-05-05
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.2.1
  downloads: 222226
  release_date: 2017-02-27
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.2.0
  downloads: 2953
  release_date: 2017-02-26
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.1.1
  downloads: 881809
  release_date: 2016-11-13
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.1.0
  downloads: 25330
  release_date: 2016-09-22
  ruby_version: ">= 0"
- !ruby/object:Gemview::Gem::Version
  version: 0.0.1
  downloads: 12314
  release_date: 2016-07-17
  ruby_version: ">= 0"
